homeassistant/components/homeassistant/triggers/homeassistant.py

```diff
@@ -46,8 +46,11 @@ def hass_shutdown(event):
                 },
                 event.context,
             )
+            if not fut_or_none:
+                return
+            hass.data.setdefault("homeassistant_triggers", []).append(fut_or_none)
```

Comment:
The magic string is not nice, just used here to quickly illustrate the concept of being able to wait only for automations triggered by shutdown.